workers/data_refinery_workers/processors/test_smasher.py

```diff
@@ -560,7 +559,7 @@ def test_no_smash_all_diff_species(self):
         experiment.accession_code = "GSE51084"
         experiment.save()
 
-        mus_mus = Organism.get_object_for_name("MUS_MUSCULUS")
+        Organism.get_object_for_name("MUS_MUSCULUS")
```

Comment:
I assume mus_mus was intended to be used here. Otherwise I can't see what this is helping to test.

Reply:
`Organism.get_object_for_name` also creates the organism in the db. Still it looks safe to delete the whole line since the organism `MUS_MUSCULUS` is not used after this.